AppCenterAuth/AppCenterAuth/Internals/Models/MSAADAuthority.m

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAADAuthority.h"
+#import "../Util/MSAuthConstants.h"
+
+@implementation MSAADAuthority
+
+static NSString *const kMSAudienceType = @"type";
```

Comment:
```suggestion
static NSString *const kMSType = @"type";
```

Reply:
Please consider to reuse common constants.

Reply:
`kMSAudienceType` and `kMSAuthorityUrl` can be moved to parent class or common constants.